ci-operator/jobs/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-release-4.15-periodics.yaml

```diff
@@ -13,6 +13,17 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-cluster-node-tuning-operator-release-4.15-e2e-nightly
+  reporter_config:
+    slack:
+      channel: '#psap-ci-alerts'
+      job_states_to_report:
+      - success
```

Comment:
I was wondering if it makes sense to report state to slack only when the state is failure or error but I can understand having a periodic green signal indicative of success can be helpful too so  LGTM.

Reply:
Absolutely. We want the know the job ran. Thank you for the review!